Description: Fix import path of square/go-jose
Author: Peter Colberg <peter@colberg.org>
Forwarded: https://github.com/hlandau/acme/pull/242
Applied-Upstream: https://github.com/hlandau/acme/commit/9cb3aa47c8786ccff014149e8db1b6b2872476f7
Last-Update: 2017-02-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/acmeapi/api_test.go
+++ b/acmeapi/api_test.go
@@ -9,7 +9,7 @@ import (
 	"fmt"
 	"github.com/hlandau/goutils/test"
 	"github.com/hlandau/xlog"
-	"github.com/square/go-jose"
+	"gopkg.in/square/go-jose.v1"
 	"golang.org/x/net/context"
 	"io/ioutil"
 	"net/http"
